server/migrations/20230921071140-rename-collection-groups.js

```diff
@@ -0,0 +1,168 @@
+"use strict";
+
+const { Op } = require("sequelize");
+
+module.exports = {
+  async up(queryInterface, Sequelize) {
+    await queryInterface.renameTable("collection_groups", "group_permissions");
```

Comment:
Lets create a view aliasing the old table name in all of these, this will allow a zero-downtime deployment.

https://brandur.org/fragments/postgres-table-rename